src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -316,6 +420,21 @@ void java_bytecode_convert_classt::convert(const classt &c)
       symbol_table,
       get_message_handler());
     method_bytecode.add(qualified_classname, method_identifier, method);
+    if(is_overlay_method(method))
```

Comment:
Again this feels like an invariant?

Reply:
Not an invariant because it depends on user data rather than on CBMC programme logic.